sark/code/function.py

```diff
@@ -86,10 +86,21 @@ def __init__(self, ea=UseCurrentAddress, name=None):
 
         elif isinstance(ea, Line):
             ea = ea.ea
-
         self._func = get_func(ea)
         self._comments = Comments(self)
 
+    @staticmethod
+    def create(ea=UseCurrentAddress, name=None):
+        try:
+            Function(ea,name)
+            raise exceptions.SarkFunctionExists("function already exists")
+        except exceptions.SarkNoFunction:
```

Comment:
I think we should an `is_function(ea)` helper, as exceptions should deal with exceptional cases, and "no function found" is the common case here.